src/Workspaces/CSharp/Portable/Extensions/ITypeSymbolExtensions.TypeSyntaxGeneratorVisitor.cs

```diff
@@ -241,10 +247,20 @@ public override TypeSyntax VisitNamespace(INamespaceSymbol symbol)
                 }
             }
 
+            private TypeSyntax AddGlobalAlias(INamespaceOrTypeSymbol symbol, SimpleNameSyntax syntax)
```

Comment:
This might be a good place to add a remark that we always add it, with the expectation the simplification system will take care of it later.